src/libs/actions/Report.js

```diff
@@ -500,9 +500,21 @@ function fetchChatReports() {
  * Get the actions of a report
  *
  * @param {Number} reportID
+ * @param {Number} [offset]
+ * @returns {Promise}
  */
-function fetchActions(reportID) {
-    API.Report_GetHistory({reportID})
+function fetchActions(reportID, offset) {
+    if (!_.isUndefined(offset) && !_.isNumber(offset)) {
+        Log.alert('[Report] Offset provided is not a number', true, {offset});
+        return;
+    }
+    const reportActionsOffset = !_.isUndefined(offset) ? offset : -1;
```

Comment:
NAB might be better to put this above the previous if block, then change that if block to just:

```javascript
if (!_.isNumber(reportActionsOffset)) {
```